doc/build-macOS.md

```diff
@@ -93,6 +93,6 @@ Other commands:
 Notes
 -----
 
-* Tested on OS X 10.8 through 10.13 on 64-bit Intel processors only.
+* Tested on OS X 10.8 Mountain Lion through macOS 10.13 High Sierra on 64-bit Intel processors only.
```

Comment:
nit: I would refer to both as macOS here

Reply:
I checked the official name for both the macOS releases and Mac OS X 10.8 Mountain Lion was renamed OS X 10.8 Mountain Lion after the macOS introduction. But I agree with you because it may improve readability.

Wikipedia for reference:
- [OS X Mountain Lion](https://en.wikipedia.org/wiki/OS_X_Mountain_Lion)
- [macOS High Sierra](https://en.wikipedia.org/wiki/MacOS_High_Sierra)